mkdocs.yml

```diff
@@ -53,6 +53,8 @@ nav:
         - Proxy: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_proxy.md
         - IP: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_ip.md
         - DNS: operation_center/integration_catalog/cloud_and_saas/cisco_umbrella/umbrella_dns.md
+      - Cloudflare:
```

Comment:
Could you also add the following entry:
```
      integrations/cloudflare.md: operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare_logpull.md
```
in the section `plugins > redirects > redirect_maps`.
Thanks